src/test/java/teammates/sqlui/webapi/DeleteAccountRequestActionTest.java

```diff
@@ -0,0 +1,132 @@
+package teammates.sqlui.webapi;
+
+import static org.mockito.Mockito.when;
+
+import java.time.Instant;
+import java.util.UUID;
+
+import org.testng.annotations.Test;
+
+import teammates.common.datatransfer.AccountRequestStatus;
+import teammates.common.util.Const;
+import teammates.storage.sqlentity.AccountRequest;
+import teammates.ui.output.MessageOutput;
+import teammates.ui.webapi.DeleteAccountRequestAction;
+import teammates.ui.webapi.InvalidOperationException;
+
+/**
+ * SUT: {@link DeleteAccountRequestAction}.
+ */
+public class DeleteAccountRequestActionTest extends BaseActionTest<DeleteAccountRequestAction> {
+
+    @Override
+    protected String getActionUri() {
+        return Const.ResourceURIs.ACCOUNT_REQUEST;
+    }
+
+    @Override
+    protected String getRequestMethod() {
+        return DELETE;
+    }
+
+    @Test
+    void testExecute_existingAccountRequest_success() {
+        AccountRequest accountRequest = new AccountRequest("email", "name", "institute", AccountRequestStatus.PENDING,
+                "comments");
+        UUID id = accountRequest.getId();
+
+        when(mockLogic.getAccountRequest(id)).thenReturn(accountRequest);
+
+        String[] params = {
+                Const.ParamsNames.ACCOUNT_REQUEST_ID, id.toString(),
```

Comment:
Good job spotting that the parameters only require the id now, instead of the email and institution! 

The old test still uses email and institution, but no need to fix it now since the test is currently disabled.